karma.webpack.config.js

```diff
@@ -0,0 +1,4 @@
+const baseConfig = require('./base.webpack.config');
+module.exports = Object.assign({}, baseConfig, {
+  devtool: 'inline-source-map',
```

Comment:
I'd suggest changing this to `cheap-module-source-map`, for faster and smaller bundles during testing.